meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+* WT: As I understood Tim, he wanted to be able to copy code snippets from specs without the need to also copy a copyright notice into the derivative work. Even snippets in the spec follows MIT, the notice needs to be included. If we want to be able to copy snippet and no notice we need ??? My question to RW is once we go from CG to WG, our specifications become W3C document. W3C license says snippets fall under W3C software license. As we transition, how can this be compatible with snippets being public domain? 
+* RW: We allow specification to have a ??? under W3C license. I'd insist we keep ??? and have a disclaimer. The only thing we need to take is notification, which is a nuisance. It's no issue to transition even if you put code snippets as being public domain. I want to prevent some big corp coming, taking the work, making incompatible changes and claiming they can. What Tim wants is being able to take snippets and put in software with no hassle, but we still should be able to say something is not compatible with what the group consensus was. You have to include an adapted MIT license, I will help you do so. In transitioning to WG, I don't see any issue.
+* WT: Even if W3C licensing document says code snippets must be under software license, we still can change that license?
+* RW: Yes.
+* SC: How do we go from what we have in the charter right now to finalizing, merging charter after making clarification? Does anyone have any objections? Especially TBL. Does what RW suggests work for you? 
+* RW: To reiterate: the plan is to make a W3C software license with no notification but a disclaimer.
+* TBL: Happy to ??? but only for the code snippets that the notification is removed. I want what the WG produces to be usable by people. 
+* WT: RW, does what we currently have in charter accomplish what you suggest?
+* RW: What is in the charter is not in the document and the object of the rule is not clear. In the charter we can give the precise URL of a specification we can make assertions. Having MIT does not do the trick because of notification. I suggest we put W3C software license and in the annex say the code snippets are under W3C license without application of notification. I may want to create a W3C snippet license that does exactly what Tim wants but this will take a long time. There's no issue whatsoever with spec authors ??? I can give you a 3-liner for the charter so contributors commit to this. 
```

Comment:
```suggestion
* RW: What is in the charter is not in the document, and the object of the rule is not clear. In the charter, we can give the precise URL of a specification; we can make assertions. Having MIT does not do the trick because of notification. I suggest we put W3C software license, and, in the annex, say the code snippets are under W3C license without application of notification. I may want to create a W3C snippet license that does exactly what Tim wants but this will take a long time. There's no issue whatsoever with spec authors ??? I can give you a 3-liner for the charter so contributors can commit to this. 
```